test/domain based tests/setCommunityStakeAmount.js

```diff
@@ -38,5 +38,11 @@ describe(scriptName, () => {
         await env.lssStaking.stakingAmount(),
       ).to.be.equal(3);
     });
+
+    it('should emit StakingAmountChanged event', async () => {
+      await expect(
+        env.lssStaking.connect(adr.lssAdmin).setStakingAmount(3),
+      ).to.emit(env.lssStaking, 'StakingAmountChanged').withArgs(3);
```

Comment:
Can we check args?

Reply:
This one is checking for the literal value 3 as the event is defined as it follows: 

`event StakingAmountChanged(uint256 indexed newAmount);`